spec/models/case/base_spec.rb

```diff
@@ -79,6 +79,47 @@
     it { should validate_presence_of(:type)          }
   end
 
+  context 'an initial transition is created after any case is create' do 
+    let(:sar_case) { create :sar_case }
+    let(:offender_sar_case) { create :offender_sar_case }
+    let(:foi_case) { create :foi_case }
+    let(:ico_foi_case) { create :ico_foi_case }
+    let(:ico_sar_case) { create :ico_sar_case }
+    let(:overturned_ico_foi) { create :overturned_ico_foi }
+    let(:overturned_ico_sar) { create :overturned_ico_sar }
+
+    it 'sar_case' do 
+      expect(sar_case.transitions.size).to eq 1
+      expect(sar_case.transitions.first.event).to eq "create"
+    end
+
+    it 'offender_sar_case' do 
+      expect(offender_sar_case.transitions.size).to eq 1
+      expect(offender_sar_case.transitions.first.event).to eq "create"
+    end
```

Comment:
There's a thing I am REALLY bad at, which is looking after tests in the same way you'd look after code. In a normal file you'd never have the same logic repeated like this, you'd make extract the behaviour into a method and DRY up the code, right? Same in tests. It can make tests harder to read but there's always a tradeoff and in this case I think we're well into the right territory for extracting this stuff into
 
1. A shared example called something like `case_has_default_transition` and your examples become more like:
```
    context 'sar_vase' do
      it_behaves_like 'case_has_default_transition'
    end
```
and probably then 2. an array of correspondence types and the above adapted to be used in a loop over them

Look through the tests for "behaves_like" to see how we've used it elsewhere and check out the docs https://relishapp.com/rspec/rspec-core/docs/example-groups/shared-examples

Reply:
@Polsonby let me re-coding this line ( actually I had a feeling you will pick up this line)